### ISSUE TITLE
usability : don't unselect text when clicking just next to the highlight color

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [burtonator/polar-bookshelf#1248]
**Describe the bug**
Something's a little annoying at the moment. If you select some text and click just next to the color you have to reselect the text again to make a hightlight.

**To Reproduce**
1. select some text
2. click on the highlight button but just a mm shy of the actual color button
3. unexpected behavior : the color popup is still here but the text has been unselected
4. you have to select again the text

**Expected behavior**
clicking on on the black part of the color popup should do nothing instead of unselecting the text

**Comment by burtonator** [comment]
ah ... interesting. so I guess you tried to select a color and then it failed?

**Comment by thiswillbeyourgithub** [comment]
No, I misclicked (clicked just a bit farther away from the actual button area).

That's part of the reason why I'm so eager to have keybindings btw.

A fix could be to make the whole black area of the popup linked to a color button, and spreading them evenly too (why is there black space at the edges ?)
